sherpa/astro/tests/test_fake_pha.py

```diff
@@ -157,24 +161,21 @@ def test_fake_pha_basic(method, is_source, expected1, expected2, has_bkg, reset_
     # For reference the predicted source signal is
     #    [200, 400, 400]
     #
-    # When is_source is not set then the values are significantly
-    # smaller as there's no response, in particular no exposure term
-    # or changes due to bin-widths varying with position.
-    #
     assert data.counts == pytest.approx(expected1)
 
     # Essentially double the exposure by having two identical arfs
     #
     data.set_arf(arf, 2)
     data.set_rmf(rmf, 2)
-    fake_pha(data, mdl, is_source=is_source, add_bkgs=False, method=method)
+    resp = data.get_full_response()
```

Comment:
Should this resp have changed compared to a few line above? If not, no harm pulling that again; I'm just wondering if the intent here is to pull compare this to the response in in line 137 to ensure that the fake_pha did not mess with it.

Reply:
So, we've just added a new pair of responses, so I think the response should be different. So we should label it differently at least. The behaviour of the new response is checked in line 173 where we check that the faked data matches the `expected2` array rather than `expected1` (see line 164). Does this help?

Reply:
Yes, that helps.